### PR TITLE
Buba the dog

### PR DESCRIPTION
**Pull request by grossir**
Review this @casey-collab

### CODE REVIEW

File: espanol/buba/buba.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Suddenly, a black dog appeared. He looked at them attentively. They could see his name
```
Comment (casey-collab) [inline espanol/buba/buba.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: espanol/pesadilla/pesadilla.md [hunk 2]
```diff
@@ -1 +1,4 @@
-¿Real o no real?
\ No newline at end of file
+¿Real o no real?
+
+[De repente, entro un perro](https://github.com/udacity/create-your-own-adventure/blob/master/espanol/buba/buba.md)
```
Comment (casey-collab) [inline espanol/pesadilla/pesadilla.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by bubaelperro** [comment]
Review this @casey-collab

File: espanol/buba/buba.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
```
Comment (casey-collab) [inline espanol/buba/buba.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**